linter/orb.yaml

```diff
@@ -16,8 +16,21 @@ commands:
         description: |
           Optional alternate config file.
         type: string
+      pre_commit_version:
+        description: |
+          Pre-commit package version.
+        type: string
```

Comment:
Looks like you might want to add `default: 'latest'`? As is, being unset would hit your second condition and attempt to `pip install pre-commit==` -- keep in mind that users can call either the `pre-commit` command (this block) _or_ the fully defined job below; both need to work!